stripe/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -323,13 +367,32 @@ class Stripe internal constructor(
         )
     }
 
+    /**
+     * Handle [next_action](https://stripe.com/docs/api/setup_intents/object#setup_intent_object-next_action)
+     * for a previously confirmed [SetupIntent]. Used for manual confirmation flow.
+     *
+     * @param activity the `Activity` that is launching the payment authentication flow
+     * @param clientSecret the [client_secret](https://stripe.com/docs/api/setup_intents/object#setup_intent_object-client_secret)
+     * property of a confirmed [SetupIntent] object
+     */
+    @UiThread
+    fun handleNextActionForSetupIntent(activity: Activity, clientSecret: String) {
+        paymentController.startAuth(
+            AuthActivityStarter.Host.create(activity),
+            clientSecret,
+            ApiRequest.Options(publishableKey, stripeAccountId)
+        )
+    }
+
     /**
      * Authenticate a [SetupIntent]. Used for manual confirmation flow.
      *
      * @param fragment the `Fragment` launching the payment authentication flow
      * @param clientSecret the [client_secret](https://stripe.com/docs/api/setup_intents/object#setup_intent_object-client_secret)
      * property of a confirmed [SetupIntent] object
      */
+    @Deprecated("Rename to better reflect behavior and match iOS naming.",
+        ReplaceWith("handleNextActionForSetupIntent(fragment, clientSecret)"))
     @UiThread
```

Comment:
not sure what the usual practice is in this case, but wouldn't it make sense to call `handleNextActionForSetupIntent` from this method to make sure they don't diverge over time?

Reply:
Same question for the other rename

Reply:
@felipecsl-stripe good point, but the methods are simple enough and the deprecated version will be dropped in an upcoming release that I'm fine with this approach.